src/main/java/enmasse/kafka/bridge/tracker/SimpleOffsetTracker.java

```diff
@@ -32,52 +32,53 @@
  */
 public class SimpleOffsetTracker<K, V> implements OffsetTracker<K, V> {
 
+	private static class PartitionState {
+		public long offset;
+		public boolean changed;
```

Comment:
In general I prefer to use private fields and getter/setter. What do you think about that ? Just to know your opinion ;)